python_ta/checkers/for_target_subscript_checker.py

```diff
@@ -0,0 +1,38 @@
+"""checker for target of for loop in subscript form.
+"""
+
+import astroid
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+from pylint.interfaces import IAstroidChecker
+
+
+class ForTargetSubscriptChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+    # name is the same as file name but without _checker part
+    name = 'for_target_subscript'
+    # use dashes for connecting words in message symbol
+    msgs = {'E9984': ('For loop variable "%s" should not be in subscript form in CSC108/CSC148.',
```

Comment:
Modify the phrasing to: `'For loop variable "%s" should not be a part of a larger object.'` This includes removing the mention of CSC108/148.